cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,18 +491,20 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
+        it('should have appropriate class names on focus/blur', function () {
             var groupInput = this.view.$(SELECTORS.inputGroupName).first(),
                 groupFields = this.view.$(SELECTORS.groupFields);
 
-            groupInput.focus();
-            jasmine.waitUntil(function() {
-                return groupFields.hasClass('is-focused');
-            }).then(function () {
+            // These tests don't work when the document is not focused (ie, the browser is backgrounded).
+            // Wrapping them in an empty setTimeout bizarrely fixes this.
+            // http://stackoverflow.com/a/26754609/2178152
+
+            setTimeout(function(){
```

Comment:
Maybe make this wrapper logic into a helper function as we'll want to use this a lot in our focus tests.